src/azure-cli/azure/cli/command_modules/security/tests/latest/test_va_sql_scenario.py

```diff
@@ -0,0 +1,218 @@
+                                       .format(resource_id, workspace_id, server_name, database_name)).get_output_in_json()
+        baseline_list_result = self.cmd('az security va sql baseline list --vm-resource-id {} --workspace-id {} --server-name {} --database-name {}'
+                                        .format(resource_id, workspace_id, server_name, database_name)).get_output_in_json()
+        self.assertEqual(scan_results_count, len(baseline_list_result["value"]))
+        self.assertEqual(baseline_set_result["value"], baseline_list_result["value"])
+
+        selected_baseline = [baseline for baseline in baseline_list_result["value"] if baseline["name"] == rule_id][0]
+        baseline_show_result = self.cmd('az security va sql baseline show --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {}'
+                                        .format(resource_id, workspace_id, server_name, database_name, rule_id)).get_output_in_json()
+        self.assertEqual(selected_baseline, baseline_show_result)
+
+        # Delete an arbitrary baseline and verify it is deleted
+        self.cmd('az security va sql baseline delete --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {}'
+                 .format(resource_id, workspace_id, server_name, database_name, rule_id))
+        _assert_error('az security va sql baseline show --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {}'
+                      .format(resource_id, workspace_id, server_name, database_name, rule_id), error_reason='no baseline set (baseline deleted)')
+
+        # Update baseline for single rule with latest results
+        baseline_update_result = self.cmd('az security va sql baseline update --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {} --latest'
+                                          .format(resource_id, workspace_id, server_name, database_name, rule_id)).get_output_in_json()
+        baseline_show_result = self.cmd('az security va sql baseline show --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {}'
+                                        .format(resource_id, workspace_id, server_name, database_name, rule_id)).get_output_in_json()
+        self.assertEqual(baseline_update_result["properties"], baseline_show_result["properties"])
+
+        # Update baseline for single rule with custom results
+        baseline_input = _get_single_baseline_input(selected_scan_result)
+        baseline_update_result = self.cmd('az security va sql baseline update --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {} {}'
+                                          .format(resource_id, workspace_id, server_name, database_name, rule_id, baseline_input)).get_output_in_json()
```

Comment:
Suggestion - For each API you should use a function that creates the command (execute the command), for example:

_build_update_baseline_command(resource_id, workspace_id, server_name, database_name, rule_id, baseline_inpu):
return 'az security va sql baseline update --vm-resource-id {} --workspace-id {} --server-name {} --database-name {} --rule-id {} {}'
                                          .format(resource_id, workspace_id, server_name, database_name, rule_id, baseline_input)